src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,5 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
```

Comment:
Btw you may have missed out the function that does the printing. You can take a look at the example given in the module website.